Sources/Purchasing/OfferingsManager.swift

```diff
@@ -366,6 +405,39 @@ private extension OfferingsManager {
             return PreviewProductType(packageType: packageType) ?? .default
         }
     }
+
+    func trackPurchaseStartedIfNeeded(trackDiagnostics: Bool) {
```

Comment:
I believe the name of the method is incorrect
```suggestion
    func trackGetOfferingsStartedIfNeeded(trackDiagnostics: Bool) {
```

Reply:
OMG I missed that, thank you! And fixed!